server/models/User.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const UserSchema = new mongoose.Schema({
+	email: {
+		type: String,
+		required: true
+	},
+	password: {
+		type: String,
+		required: true
+	},
+	my_lists: {
+		type: Array
```

Comment:
Shouldn't this be a reference ID to List Schema? Because we have a different Schema for lists and products eh?

Reply:
Yes, I also had meant to ask about that before I got glitched out haha! How would I indicate that on the schema? Just as a string to store an ID in? If so would it make sense to change the prop name to my_lists_id?

Reply:
maybe something like `listIds : [{type: mongoose.Schema.Types.ObjectId, ref: 'List'}]`

Reply:
Yes that is correct @sina-jamshidi. No worries anyways our schema would be changing as per requirements!